### PR TITLE
(SERVER-2399) Mention JAVA_ARGS_CLI wrt changing tmpdir

### PR DESCRIPTION
**Pull request by justinstoller**

### CODE REVIEW

File: documentation/known_issues.markdown [hunk 1]
```diff
@@ -43,9 +43,11 @@ To use a different temporary directory, you can set the following JVM property:
 ```
 
 When Puppet Server is installed from packages, this property should be added
-to the `JAVA_ARGS` variable defined in either `/etc/sysconfig/puppetserver`
-or `/etc/default/puppetserver`, depending on upon your distribution. Note that
-the service will need to be restarted in order for this change to take effect.
+to the `JAVA_ARGS` and `JAVA_ARGS_CLI` variables defined in either
+`/etc/sysconfig/puppetserver` or `/etc/default/puppetserver`, depending on
+upon your distribution. Invocations of the `gem`, `ruby`, and `irb` subcommands
```
Comment (clairecadman) [inline documentation/known_issues.markdown]: Omit "upon".

File: documentation/known_issues.markdown [hunk 2]
```diff
@@ -43,9 +43,11 @@ To use a different temporary directory, you can set the following JVM property:
 ```
 
 When Puppet Server is installed from packages, this property should be added
```
Comment (clairecadman) [inline documentation/known_issues.markdown]: Suggested reword: "When Puppet Server is installed from packages, add this property to the...."

File: documentation/known_issues.markdown [hunk 3]
```diff
@@ -43,9 +43,11 @@ To use a different temporary directory, you can set the following JVM property:
...
-to the `JAVA_ARGS` variable defined in either `/etc/sysconfig/puppetserver`
-or `/etc/default/puppetserver`, depending on upon your distribution. Note that
-the service will need to be restarted in order for this change to take effect.
+to the `JAVA_ARGS` and `JAVA_ARGS_CLI` variables defined in either
+`/etc/sysconfig/puppetserver` or `/etc/default/puppetserver`, depending on
+upon your distribution. Invocations of the `gem`, `ruby`, and `irb` subcommands
+will use the updated `JAVA_ARGS_CLI` on their next invocation. The service will
```
Comment (clairecadman) [inline documentation/known_issues.markdown]: Omit the first "will".

**Comment by justinstoller** [comment]
Much better, thanks for the proof read, Claire.

**Pull request merged**